libs/s25main/world/GameWorld.cpp

```diff
@@ -779,6 +779,11 @@ void GameWorld::Attack(const unsigned char player_attacker, const MapPoint pt, c
     // Send the soldiers to attack
     unsigned curNumSoldiers = 0;
 
+    if(!potentialAttackers.empty() && HasLua())
+    {
+        GetLua().EventAttack(player_attacker, attacked_building->GetPlayer());
+    }
```

Comment:
maybe It'd be cool to have the number of attackers as well, since we have the count later anyway.

Reply:
Good improvement. Added additional parameters for the number of attackers.